src/main/kotlin/no/skatteetaten/aurora/boober/feature/Feature.kt

```diff
@@ -143,9 +146,9 @@ val AuroraDeploymentSpec.applicationPlatform: ApplicationPlatform get() = this["
 class HeaderHandlers private constructor(defaultAppName: String, defaultEnvName: String) {
```

Comment:
Gjerne sjekk at allowedFilesTypes er riktig for feltene. Vi har lagt til allowedFilesTypes på mer enn bare envpointers